File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCS0503Q00LoadConsultInfo.java
package nta.med.data.model.ihis.ocsa;

import java.sql.Timestamp;

public class OCS0503Q00LoadConsultInfo{
	private String ioGubun;
    private String answerYn;        
    private String gwaConsultYn;
    private Timestamp naewonDate;
    private String jinryoTime ;
    private String bunho   ;
    private String suname ;
    private String suname2 ;
    private String hoDong ;
    private String sexAge;
    private String gubunName;
    private String gwa;
    private String doctor;
    private String naewonType;
    private Double jubsuNo;
    private String chojae;
    private String chojaeName;
    private Double pkNaewon;
    private String naewonYn;
    private Timestamp reqDate;
    private String reqGwa;
    private String reqDoctor;
    private String consultGwa;
    private String consultDoctor;
    private String reqIoGubun;
    private String gwaName;
    private String doctorName;
    private String answerUpdYn;
    private Timestamp appDate;
    private Double pkocs0503;
    private String orderEndYn;
	public OCS0503Q00LoadConsultInfo(String ioGubun, String answerYn,
			String gwaConsultYn, Timestamp naewonDate, String jinryoTime,
			String bunho, String suname, String suname2, String hoDong,
			String sexAge, String gubunName, String gwa, String doctor,
			String naewonType, Double jubsuNo, String chojae,
			String chojaeName, Double pkNaewon, String naewonYn,
			Timestamp reqDate, String reqGwa, String reqDoctor,
			String consultGwa, String consultDoctor, String reqIoGubun,
			String gwaName, String doctorName, String answerUpdYn,
			Timestamp appDate, Double pkocs0503, String orderEndYn) {
		super();
		this.ioGubun = ioGubun;
		this.answerYn = answerYn;
		this.gwaConsultYn = gwaConsultYn;
		this.naewonDate = naewonDate;
		this.jinryoTime = jinryoTime;
		this.bunho = bunho;
		this.suname = suname;
		this.suname2 = suname2;
		this.hoDong = hoDong;
		this.sexAge = sexAge;
		this.gubunName = gubunName;
		this.gwa = gwa;
		this.doctor = doctor;
		this.naewonType = naewonType;
		this.jubsuNo = jubsuNo;
		this.chojae = chojae;
		this.chojaeName = chojaeName;
		this.pkNaewon = pkNaewon;
		this.naewonYn = naewonYn;
		this.reqDate = reqDate;
		this.reqGwa = reqGwa;
		this.reqDoctor = reqDoctor;
		this.consultGwa = consultGwa;
		this.consultDoctor = consultDoctor;
		this.reqIoGubun = reqIoGubun;
		this.gwaName = gwaName;
		this.doctorName = doctorName;
		this.answerUpdYn = answerUpdYn;
		this.appDate = appDate;
		this.pkocs0503 = pkocs0503;
		this.orderEndYn = orderEndYn;
	}
	public String getIoGubun() {
		return ioGubun;
	}
	public void setIoGubun(String ioGubun) {
		this.ioGubun = ioGubun;
	}
	public String getAnswerYn() {
		return answerYn;
	}
	public void setAnswerYn(String answerYn) {
		this.answerYn = answerYn;
	}
	public String getGwaConsultYn() {
		return gwaConsultYn;
	}
	public void setGwaConsultYn(String gwaConsultYn) {
		this.gwaConsultYn = gwaConsultYn;
	}
	public Timestamp getNaewonDate() {
		return naewonDate;
	}
	public void setNaewonDate(Timestamp naewonDate) {
		this.naewonDate = naewonDate;
	}
	public String getJinryoTime() {
		return jinryoTime;
	}
	public void setJinryoTime(String jinryoTime) {
		this.jinryoTime = jinryoTime;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getSuname() {
		return suname;
	}
	public void setSuname(String suname) {
		this.suname = suname;
	}
	public String getSuname2() {
		return suname2;
	}
	public void setSuname2(String suname2) {
		this.suname2 = suname2;
	}
	public String getHoDong() {
		return hoDong;
	}
	public void setHoDong(String hoDong) {
		this.hoDong = hoDong;
	}
	public String getSexAge() {
		return sexAge;
	}
	public void setSexAge(String sexAge) {
		this.sexAge = sexAge;
	}
	public String getGubunName() {
		return gubunName;
	}
	public void setGubunName(String gubunName) {
		this.gubunName = gubunName;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
	public String getDoctor() {
		return doctor;
	}
	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}
	public String getNaewonType() {
		return naewonType;
	}
	public void setNaewonType(String naewonType) {
		this.naewonType = naewonType;
	}
	public Double getJubsuNo() {
		return jubsuNo;
	}
	public void setJubsuNo(Double jubsuNo) {
		this.jubsuNo = jubsuNo;
	}
	public String getChojae() {
		return chojae;
	}
	public void setChojae(String chojae) {
		this.chojae = chojae;
	}
	public String getChojaeName() {
		return chojaeName;
	}
	public void setChojaeName(String chojaeName) {
		this.chojaeName = chojaeName;
	}
	public Double getPkNaewon() {
		return pkNaewon;
	}
	public void setPkNaewon(Double pkNaewon) {
		this.pkNaewon = pkNaewon;
	}
	public String getNaewonYn() {
		return naewonYn;
	}
	public void setNaewonYn(String naewonYn) {
		this.naewonYn = naewonYn;
	}
	public Timestamp getReqDate() {
		return reqDate;
	}
	public void setReqDate(Timestamp reqDate) {
		this.reqDate = reqDate;
	}
	public String getReqGwa() {
		return reqGwa;
	}
	public void setReqGwa(String reqGwa) {
		this.reqGwa = reqGwa;
	}
	public String getReqDoctor() {
		return reqDoctor;
	}
	public void setReqDoctor(String reqDoctor) {
		this.reqDoctor = reqDoctor;
	}
	public String getConsultGwa() {
		return consultGwa;
	}
	public void setConsultGwa(String consultGwa) {
		this.consultGwa = consultGwa;
	}
	public String getConsultDoctor() {
		return consultDoctor;
	}
	public void setConsultDoctor(String consultDoctor) {
		this.consultDoctor = consultDoctor;
	}
	public String getReqIoGubun() {
		return reqIoGubun;
	}
	public void setReqIoGubun(String reqIoGubun) {
		this.reqIoGubun = reqIoGubun;
	}
	public String getGwaName() {
		return gwaName;
	}
	public void setGwaName(String gwaName) {
		this.gwaName = gwaName;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	public String getAnswerUpdYn() {
		return answerUpdYn;
	}
	public void setAnswerUpdYn(String answerUpdYn) {
		this.answerUpdYn = answerUpdYn;
	}
	public Timestamp getAppDate() {
		return appDate;
	}
	public void setAppDate(Timestamp appDate) {
		this.appDate = appDate;
	}
	public Double getPkocs0503() {
		return pkocs0503;
	}
	public void setPkocs0503(Double pkocs0503) {
		this.pkocs0503 = pkocs0503;
	}
	public String getOrderEndYn() {
		return orderEndYn;
	}
	public void setOrderEndYn(String orderEndYn) {
		this.orderEndYn = orderEndYn;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS0503Q00LoadConsultInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
	public class OCS0503Q00LoadConsultInfo
	{
		private String _ioGubun;
		private String _answerYn;
		private String _gwaConsultYn;
		private String _naewonDate;
		private String _jinryoTime;
		private String _bunho;
		private String _suname;
		private String _suname2;
		private String _hoDong;
		private String _sexAge;
		private String _gubunName;
		private String _gwa;
		private String _doctor;
		private String _naewonType;
		private String _jubsuNo;
		private String _chojae;
		private String _chojaeName;
		private String _pkNaewon;
		private String _naewonYn;
		private String _reqDate;
		private String _reqGwa;
		private String _reqDoctor;
		private String _consultGwa;
		private String _consultDoctor;
		private String _reqIoGubun;
		private String _consGwaName;
		private String _consDoctorName;
		private String _answerUpdYn;
		private String _appDate;
		private String _pkocs0503;
		private String _orderEndYn;

		public String IoGubun
		{
			get { return this._ioGubun; }
			set { this._ioGubun = value; }
		}

		public String AnswerYn
		{
			get { return this._answerYn; }
			set { this._answerYn = value; }
		}

		public String GwaConsultYn
		{
			get { return this._gwaConsultYn; }
			set { this._gwaConsultYn = value; }
		}

		public String NaewonDate
		{
			get { return this._naewonDate; }
			set { this._naewonDate = value; }
		}

		public String JinryoTime
		{
			get { return this._jinryoTime; }
			set { this._jinryoTime = value; }
		}

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String Suname
		{
			get { return this._suname; }
			set { this._suname = value; }
		}

		public String Suname2
		{
			get { return this._suname2; }
			set { this._suname2 = value; }
		}

		public String HoDong
		{
			get { return this._hoDong; }
			set { this._hoDong = value; }
		}

		public String SexAge
		{
			get { return this._sexAge; }
			set { this._sexAge = value; }
		}

		public String GubunName
		{
			get { return this._gubunName; }
			set { this._gubunName = value; }
		}

		public String Gwa
		{
			get { return this._gwa; }
			set { this._gwa = value; }
		}

		public String Doctor
		{
			get { return this._doctor; }
			set { this._doctor = value; }
		}

		public String NaewonType
		{
			get { return this._naewonType; }
			set { this._naewonType = value; }
		}

		public String JubsuNo
		{
			get { return this._jubsuNo; }
			set { this._jubsuNo = value; }
		}

		public String Chojae
		{
			get { return this._chojae; }
			set { this._chojae = value; }
		}

		public String ChojaeName
		{
			get { return this._chojaeName; }
			set { this._chojaeName = value; }
		}

		public String PkNaewon
		{
			get { return this._pkNaewon; }
			set { this._pkNaewon = value; }
		}

		public String NaewonYn
		{
			get { return this._naewonYn; }
			set { this._naewonYn = value; }
		}

		public String ReqDate
		{
			get { return this._reqDate; }
			set { this._reqDate = value; }
		}

		public String ReqGwa
		{
			get { return this._reqGwa; }
			set { this._reqGwa = value; }
		}

		public String ReqDoctor
		{
			get { return this._reqDoctor; }
			set { this._reqDoctor = value; }
		}

		public String ConsultGwa
		{
			get { return this._consultGwa; }
			set { this._consultGwa = value; }
		}

		public String ConsultDoctor
		{
			get { return this._consultDoctor; }
			set { this._consultDoctor = value; }
		}

		public String ReqIoGubun
		{
			get { return this._reqIoGubun; }
			set { this._reqIoGubun = value; }
		}

		public String ConsGwaName
		{
			get { return this._consGwaName; }
			set { this._consGwaName = value; }
		}

		public String ConsDoctorName
		{
			get { return this._consDoctorName; }
			set { this._consDoctorName = value; }
		}

		public String AnswerUpdYn
		{
			get { return this._answerUpdYn; }
			set { this._answerUpdYn = value; }
		}

		public String AppDate
		{
			get { return this._appDate; }
			set { this._appDate = value; }
		}

		public String Pkocs0503
		{
			get { return this._pkocs0503; }
			set { this._pkocs0503 = value; }
		}

		public String OrderEndYn
		{
			get { return this._orderEndYn; }
			set { this._orderEndYn = value; }
		}

		public OCS0503Q00LoadConsultInfo() { }

		public OCS0503Q00LoadConsultInfo(String ioGubun, String answerYn, String gwaConsultYn, String naewonDate, String jinryoTime, String bunho, String suname, String suname2, String hoDong, String sexAge, String gubunName, String gwa, String doctor, String naewonType, String jubsuNo, String chojae, String chojaeName, String pkNaewon, String naewonYn, String reqDate, String reqGwa, String reqDoctor, String consultGwa, String consultDoctor, String reqIoGubun, String consGwaName, String consDoctorName, String answerUpdYn, String appDate, String pkocs0503, String orderEndYn)
		{
			this._ioGubun = ioGubun;
			this._answerYn = answerYn;
			this._gwaConsultYn = gwaConsultYn;
			this._naewonDate = naewonDate;
			this._jinryoTime = jinryoTime;
			this._bunho = bunho;
			this._suname = suname;
			this._suname2 = suname2;
			this._hoDong = hoDong;
			this._sexAge = sexAge;
			this._gubunName = gubunName;
			this._gwa = gwa;
			this._doctor = doctor;
			this._naewonType = naewonType;
			this._jubsuNo = jubsuNo;
			this._chojae = chojae;
			this._chojaeName = chojaeName;
			this._pkNaewon = pkNaewon;
			this._naewonYn = naewonYn;
			this._reqDate = reqDate;
			this._reqGwa = reqGwa;
			this._reqDoctor = reqDoctor;
			this._consultGwa = consultGwa;
			this._consultDoctor = consultDoctor;
			this._reqIoGubun = reqIoGubun;
			this._consGwaName = consGwaName;
			this._consDoctorName = consDoctorName;
			this._answerUpdYn = answerUpdYn;
			this._appDate = appDate;
			this._pkocs0503 = pkocs0503;
			this._orderEndYn = orderEndYn;
		}

	}
}